web/concrete/src/Page/Type/Type.php

```diff
@@ -1102,19 +1102,14 @@ public function getPageTypeValidatorObject()
         }
     }
 
-    public function createDraft(PageTemplate $pt, $u = false)
+    public function createDraft(PageTemplate $pt)
```

Comment:
any calls to this method with a user supplied will cause errors with this default value removed